toolchain/syno-armv7-1.3/Makefile

```diff
@@ -0,0 +1,19 @@
+TC_NAME = syno-armv7
+
+# generic toolchain for arm7hf SRM
+TC_ARCH = ipq806x dakota
```

Comment:
Shouldn't it also contain all archs such as: `northstarplus ipq806x dakota hawkeye` ?

Reply:
northstarplus is not supported (limited to SRM 1.2)
and hawkeye is _a mess_: the cpu is aarch64 but the toolchain is binary the same as dakota, therefor I didn't add it to a generic arch yet.

Reply:
Would you mind adding a comment?  at least for hawkeye to avoid confusion later-on?